### PR TITLE
Directories definition

### LINKED ISSUE #93: Directories Definition
definition in Section 3

`Directories (also called "folders") allow for the organization of files into tree-like hierarchies.`

### PR DESCRIPTION
**Pull request by rosy1280**
Fixes #93

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -173,8 +173,14 @@ <h2>Terminology</h2>
...
-        <dt><dfn>Directories</dfn>:</dt>
-        <dd></dd>
+        <dt><dfn id="dfn-directories">Directories</dfn>:</dt>
+        <dd>Directories (also called "folders") allow for the organization of files
+          into tree-like hierarchies. Directories are used to organize <a
+          href="#dfn-ocfl-object">OCFL Objects</a> into <a
+          href="#dfn-version">Versions</a>.  Alternatively they may be used to organize
+          the content of an OCFL Object itself. Empty directories within a version
+          directory are not permitted. However empty directories may be maintained by
+          creating a .keep file within that directory.</dd>
```
Comment (ahankinson) [inline index.html]: should be wrapped in `<code>` tags.

**Comment by zimeon** [comment]
IMO, we don't need an entry for Directory (or File) in the terminology section. These aren't terms that we introduce or assert to have special meaning other than the normal one

**Comment by rosy1280** [comment]
F2F 2018.09.07: decision to remove the directory definition and the file definition from the terminology section.  we should discuss them in the introduction.

**Comment by rosy1280** [comment]
Fixes #92 as well

**Comment by rosy1280** [comment]
Fixes #92

**Pull request merged**